api/lib/api/clients/common.ex

```diff
@@ -0,0 +1,33 @@
+defmodule Tesla.Middleware.RequestLogger do
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Modules should have a @moduledoc tag.](https://app.codacy.com/manual/lucaskjaero/foreign-language-reader/pullRequest?prid=5112709)